pkg/cmd/cli/backuplocation/create.go

```diff
@@ -148,6 +152,13 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		validationFrequency = &metav1.Duration{Duration: o.ValidationFrequency}
 	}
 
+	var secretName, secretKey string
+	for k, v := range o.Credential.Data() {
```

Comment:
Should we also make sure that the `len(o.Credential.Data()) == 1` here? That way we ensure there's no surprise if a user had multiple secret files in the Kubernetes Secret, we're making sure up front we're given 1 and only 1 option.

The help text says that only 1 will be used.